### PR TITLE
Adding Token Detection Toggle to Security & Privacy

### PR DESCRIPTION
**Pull request by NiranjanaBinoy**
This PR is a part of issue #11619.

The PR adds a Token Detection Toggle to the Security & Privacy

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -2648,6 +2648,12 @@
   "usePhishingDetectionDescription": {
     "message": "Display a warning for phishing domains targeting Ethereum users"
   },
+  "useStaticTokenList": {
+    "message": "Token detection"
+  },
+  "useStaticTokenListDescription": {
+    "message": "We use third-party APIs to detect and display new tokens sent to your wallet. Turn off if you don’t want the app to pull data from those those services."
```
Comment (ryanml) [inline app/_locales/en/messages.json:2655]: nit: Just a suggestion, but maybe we could adjust the wording to `We use third-party APIs to detect and display new tokens sent to your wallet. Turn this off if you don’t want MetaMask to pull data from those those services.`?

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -2648,6 +2648,12 @@
   "usePhishingDetectionDescription": {
     "message": "Display a warning for phishing domains targeting Ethereum users"
   },
+  "useStaticTokenList": {
+    "message": "Token detection"
```
Comment (darkwing) [inline app/_locales/en/messages.json:2652]: Each other setting in this panel starts with an action word, so I think we should change this to "Use Token Detection"

**Review by ryanml — DISMISSED** [review]
lgtm!

File: app/_locales/en/messages.json [hunk 3]
```diff
@@ -2648,6 +2648,12 @@
   "usePhishingDetectionDescription": {
     "message": "Display a warning for phishing domains targeting Ethereum users"
   },
+  "useStaticTokenList": {
+    "message": "Use Token Detection"
```
Comment (adonesky1) [inline app/_locales/en/messages.json:2652]: Doesn't `useStaticTokenList` mean the opposite of `Use Token Detection`?

Comment (NiranjanaBinoy) [reply app/_locales/en/messages.json:2652]: The controller flag that will be updated with the toggle button is named `useStaticTokenList`, so I went with the same name for the key.

**Review by adonesky1 — DISMISSED** [review]
LGTM!

File: app/_locales/en/messages.json (hunk 3 above)
Comment (adonesky1) [reply app/_locales/en/messages.json:2652]: Per our conversation on #11398 I would vote to change this to `useTokenDetection`!

Comment (NiranjanaBinoy) [reply app/_locales/en/messages.json:2652]: This will be updated after #11398 is merged

**Review by mcmire — APPROVED** [review]
Seems straightforward!

**Review by danjm — APPROVED** [review]
LGTM

**Pull request merged**